jacoco-maven-plugin.test/it/it-report-aggregate/invoker.properties

```diff
@@ -0,0 +1 @@
+invoker.goals = -T2 clean install
```

Comment:
IMO approach based on dependencies does not require special testing of parallel build by definition. Parallel builds are available only since Maven 3.x , so that this test can't be executed with Maven 2.x.